GPU/GPUTracking/DataTypes/GPUTRDTrack.cxx

```diff
@@ -31,6 +31,7 @@ GPUd() void GPUTRDTrack_t<T>::initialize()
 {
   // set all members to their default values (needed since in-class initialization not possible with AliRoot)
   mChi2 = 0.f;
+  mIsPadrowCrossing = 0;
```

Comment:
There are more functions in `GPUTRDTrack` where the `mPadRowCrossing` variable should be initialized. It should in addition be added to `GPU/GPUTracking/TRDTracking/GPUTRDTrackData.h`